Docs: Update enter/shift+enter details for prometheus query field (#52607)
grafana/grafana: docs/sources/explore/query-management.md

### Query field

> The Query field supports autocomplete for metric names, function and works mostly the same way as the standard Prometheus query editor. Press the enter key to create a new line, and shift+enter to execute a query.

Reviewer: Does the autocomplete work for metric names and functions? Or does the sentence mean that functions and usage are mostly the same as the Prometheus query editor. If that's the case, here's a suggestion. What do you think? 
```suggestion
The Query field supports autocomplete for metric names. The editor has functions comparable to the standard Prometheus query editor. Press the Enter key to create a new line and Shift+Enter to execute a query.
```

Author: @knylander-grafana I think metric names and functions are equivalent in this sentence. What do you think about "The Query field supports autocomplete for metric names and functions, comparable to the standard Prometheus query editor."

Reviewer: That sounds great!
